src/components/data-table-item/data-table-item.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./data-table-item.scss";
+
+export const DataTableItem = ({ content, align, extraStyles }) => {
+  return (
+    <div
+      className={`data-table-item data-table-item--${align}`}
+      style={extraStyles}
+    >
+      {content}
+    </div>
+  );
+};
+
+DataTableItem.propTypes = {
+  content: PropTypes.any,
```

Comment:
I believe a better solution for this is to use the `children` prop as you did for the row component.